src/app/containers/CpsOnwardJourney/index.jsx

```diff
@@ -116,6 +117,35 @@ const SingleContentWrapper = styled.div`
   `}
 `;
 
+const OptionallyRenderedSkipWrapper = ({ skipLink, service, children }) =>
+  skipLink ? (
+    <SkipLinkWrapper service={service} {...skipLink}>
+      {children}
+    </SkipLinkWrapper>
+  ) : (
+    children
+  );
+
+const optionalSkipLinkProps = {
```

Comment:
I realise "optional" is coming from the name of the component, but these props are not optional. Can I suggest we rename this variable to `skipLinkProps`?
You also might opt to call it `skipLinkShape` and also include the shared propType function.